src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -28,6 +28,12 @@ namespace System.Management.Automation
     /// </summary>
     internal class MshCommandRuntime : ICommandRuntime2
     {
+        internal const string WriteErrorStreamPropertyName = "ErrorStreamWrite";
+        internal const string WriteWarningStreamPropertyName = "WarningStreamWrite";
```

Comment:
This property is also checked in the console host.

But I think it's a bad idea to change it, it might affect cross-version remoting.

Reply:
I can to the same length check as you suggested above if it matters performance wise.